nncf/torch/graph/pattern_operations.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2020-2021 Intel Corporation.
+#
+# LEGAL NOTICE: Your use of this software and any required dependent software
+# (the "Software Package") is subject to the terms and conditions of
+# the Intel(R) OpenVINO(TM) Distribution License for the Software Package,
+# which may also include notices, disclaimers, or license terms for
+# third party or open source software included in or with the Software Package,
+# and your use indicates your acceptance of all such terms. Please refer
+# to the "third-party-programs.txt" or other similarly-named text file
+# included with the Software Package for additional details.
```

Comment:
@vshampor, @MaximProshin Is it correct license?

Reply:
Most likely not. @aleksei-kashapov use the Apache license as is being done in the majority of the other files.

Reply:
Updated